backend/src/main/java/saci/domain/service/LevelService.java

```diff
@@ -34,10 +26,33 @@ public Level createLevel(Level level) {
             throw new CoefficientOverlapException(
                     "Error Creating the Level: Overlapping Coefficients or Invalid Coefficients");
         }
+
         Optional<Level> optionalLevel = levelRepository.findByName(level.getName());
         if (optionalLevel.isPresent()) {
             throw new AlreadyExistsException("Level name already exists");
         }
+
         return levelRepository.save(level);
     }
+
+    public List<Level> getLevels() {
```

Comment:
A scenario where we need to retrieve all levels, without filter based on role doesn`t exist. Please, get rid of this method